.pre-commit-hooks.yaml

```diff
@@ -1,6 +1,5 @@
 - id: jwt-checker
   name: jwt-checker
-  description: Check if JWT token will be upload
-  entry: python jwt-checker.py
-  language: system
-  stage: [commit]
+  description: Check to avoid uploading JWT tokens
+  entry: jwt-checker
```

Comment:
I _believe_ this entry will always try to run local files in the target repo, not in the hook repo. So instead of passing the script to run we are using a command in the hook virtual environment.